.travis.yml

```diff
@@ -1,9 +1,16 @@
 language: php
 
 php:
-  - 5.3
   - 5.4
   - 5.5
+  - 5.6
+  - 7
+  - hhvm
+
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
The tests were failing for php 7 (fixed here https://github.com/davidwdan/Rx.PHP/commit/13a2534bbb97d1b075f00fbbfc93d2670bb2f2ed) and since it's not released yet, I figured it'd probably be better to allow failures.